### PR TITLE
feat(ONYX-742): notifications empty state redesign

### PR DESCRIPTION
**Pull request by tam-kis**
The type of this PR is: Feat

This PR solves [ONYX-742](https://artsyproduct.atlassian.net/browse/ONYX-742)

### Description
Update Notifications empty state design, added links "Artists" and "Galleries" that lead to their pages.

https://github.com/artsy/force/assets/17580625/5badc5dd-2329-44f8-8d04-ccf05784ed1f

https://github.com/artsy/force/assets/17580625/0b3a7ad3-bee5-4ff2-88b8-89e9cddd18ee

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ONYX-742]: https://artsyproduct.atlassian.net/browse/ONYX-742?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/Notifications/__tests__/NotificationsEmptyStateByType.jest.tsx [hunk 1]
```diff
@@ -13,9 +13,9 @@ describe("NotificationsEmptyStateByType", () => {
   it("should render correct state when type is 'All'", () => {
     render(<NotificationsEmptyStateByType type="all" />)
 
-    const title = "Follow artists and galleries to stay up to date"
+    const title = "Stay up to date with the artists and artworks you love"
```
Comment (MounirDhahri) [inline src/Components/Notifications/__tests__/NotificationsEmptyStateByType.jest.tsx:16]:
```suggestion
    const { title, message } = emptyStateByType("all")
```

**Review by MounirDhahri — APPROVED** [review]
LGTM!

File: src/Components/Notifications/NotificationsContextualMenu.tsx [hunk 2]
```diff
@@ -21,13 +20,13 @@ export const NotificationsContextualMenu: React.FC<NotificationsContextualMenuPr
...
           onClick={onHide}
-          to={BASE_SAVES_PATH}
+          to={"/settings/alerts"}
           textDecoration="none"
           color="black100"
           display="block"
           p={2}
         >
-          Manage Saves
+          Manage Alerts
```
Comment (tam-kis) [inline src/Components/Notifications/NotificationsContextualMenu.tsx:29]: Update copy and link in ContextualModal

File: src/Components/Notifications/NotificationsPills.tsx [hunk 3]
```diff
@@ -19,16 +19,12 @@ export const NotificationsPills: React.FC = () => {
...
-  const hasFollowNotifications =
-    extractNodes(data?.viewer?.followNotifications).length > 0
 
   const notificationPills = compact([
     { value: "All", name: "all" },
     hasPartnerOfferNotifications && { value: "Offers", name: "offers" },
-    hasAlertNotifications && { value: "Alerts", name: "alerts" },
-    hasFollowNotifications && { value: "Following", name: "following" },
+    { value: "Alerts", name: "alerts" },
+    { value: "Following", name: "following" },
```
Comment (tam-kis) [inline src/Components/Notifications/NotificationsPills.tsx:27]: "Alerts" and "Following" filters should be visible even for empty state. Only "Offers" is contitional.

**Review by MounirDhahri — APPROVED** [review]
thanks for the update. I guess we don't need something for the offer pills?

**Comment by tam-kis** [comment]
@MounirDhahri The "Offer" pill is conditional so when there is an empty state - it is not visible, unlike "Alerts" and "Follows"

**Comment by MounirDhahri** [comment]
Perfect 👌

**Pull request merged**